packages/ui-library/src/elements/toast/style.css

```diff
@@ -0,0 +1,12 @@
+@layer components {
+    .yst-root {
+
+        .yst-toast {
```

Comment:
Not a big deal, but I would consider placing every CSS rule in a different line for readability. E.g.

    .yst-toast {
            @apply
			yst-w-80
			yst-max-w-full
			yst-overflow-y-auto
			yst-p-4
			yst-bg-white
			yst-shadow-lg y
			st-rounded-lg
			yst-ring-1
			yst-ring-black
			yst-ring-opacity-5
			yst-z-20
			yst-pointer-events-auto;
        }

        `.yst-toast--large {
            @apply
	              yst-w-96;
        }`

I am also looking at the stylesheets of other elements/components in the UI library, where we also do it this way.

Reply:
I think that the general approach is to keep the `@apply` on the same line as the Tailwind classes when they are a few (like 2 or 3), so I'm updating just the `.yst-toast` one 🙂